products/memgraph/upcoming/tutorials/replication.md

```diff
@@ -0,0 +1,135 @@
+## How to Set Up Replication on a Small Cluster (Enterprise)
+
+This article is a part of a series intended to showcase Memgraph's features,
+and bring the user up to speed on developing with Memgraph.
+
+We highly recommend checking out the other articles from this series which
+are listed in our [tutorial overview section](tutorials.md).
+
+### Introduction
+
+This example demonstrates how to create a simple cluster of nodes running
+Memgraph instances, and set up replication. We'll demonstrate the use and
+behavior of the available synchronization modes.
+
+### Cluster topology
+
+The cluster consists of four nodes, a single main and three replicas:
+
+* the main node, containing the original data to be replicated to other nodes,
+* a node that will be replicated to using the sync mode,
+* a node that will be replicated to using the async mode,
+* a node that will be replicated to using the sync-with-timeout mode.
+
+We'll be hitting the main node with create queries, creating nodes and edges,
+while running BFS on the replicas, and observing the effects of different sync
+modes used.
+
+### Configuring the cluster
+
+We'll use Docker to set up and run the cluster on your local machine, so make
+sure you have it installed and ready, and grab your Memgraph docker image.
+Look [here](https://docs.memgraph.com/memgraph/getting-started/installation/docker-installation) for instructions.
+In addition, make sure you have either [mgclient](https://github.com/memgraph/mgclient) or [mgconsole](https://github.com/memgraph/mgconsole) installed.
```

Comment:
I would write this differently.
We assume you have already setup a client for running queries like mgconsole.
You can also use any of the supported drivers like mgclient or any of the Neo4j drivers. (Neo4j part being optional)

Reply:
Sure, sounds better.